backends/tfhe-cuda-backend/cuda/src/pbs/programmable_bootstrap_amortized.cuh

```diff
@@ -89,7 +89,9 @@ __global__ void device_programmable_bootstrap_amortized(
 
   // Put "b", the body, in [0, 2N[
   Torus b_hat = 0;
-  modulus_switch(block_lwe_array_in[lwe_dimension], b_hat,
+  Torus e_mms = centered_binary_modulus_switch_body_correction_to_add(
```

Comment:
Do you want to hardcode the new MS technique here ?

Comment applies to other places

Reply:
I refactored my implementation and now we have a switch. It selects between no technique to reduce noise, drift, or the centered technique. It requires changing a lot of places, so the PR became much bigger. Sorry about that.

cc @agnesLeroy